src/components/team_sidebar/TeamSidebar.scss

```diff
@@ -86,10 +85,10 @@
     background-color: transparent;
     transition: opacity .1s, background-color .1s;
     outline: 0;
-    opacity: .5;
+    opacity: .6;
```

Comment:
really not a big fan of color selection via opacity. Can't we just set a specific gray? Or is it intentional to see the images below now, that'd be fine with me then.